src/ProjectConfiguration.ts

```diff
@@ -376,6 +390,32 @@ export function validateDefaultLanguage(config: ProjectConfiguration) {
   return diagnostics;
 }
 
+export function validateStorageGroup(config: ProjectConfiguration) {
+  const diagnostics = new DiagnosticList();
+  const permissionKey = 'access_app_cluster_storage';
```

Comment:
I see we reuse this key. Move it to the top in a const?

Reply:
Changed permissions to a string enum rather than do a one-off for this.